basicsynbio/cam/echo_instructions.py

```diff
@@ -30,6 +37,12 @@ def export_echo_assembly(
         buffer_well (optional): location in 6 well plate of assembly buffer.
         water_well (optional): location in 6 well plate of dH20.
         alternate_well (optional): specifies whether alternating wells are to be used in the input 384 well plate.
+        assemblies_per_clip (optional): specifies the number of clip reactions (to create assemblies) one filled
```

Comment:
We can use the same description for the assemblies_per_clip argument as used in the pdf_instructions function.

Reply:
copy and pasted the one from pdf_instructions over 👍